spec/environment_manager_spec.rb

```diff
@@ -22,6 +22,13 @@
     expect(e.ci_source.pull_request_id).to eq(number.to_s)
   end
 
+  it 'stores buildkite in the source' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.